doc/howtos/booting-and-installation.rst

```diff
@@ -41,13 +41,18 @@ USB thumb drive or SD card.  The typical way to do this is with the :command:`dd
 
      $ dmesg 
 
-   or you can use the :command:`lsblk` command to show the block-level devices; a USB drive usually shows up as ``/sdb`` or ``/sdc``
-   (almost never as ``/sda``)
+   or you can use the :command:`lsblk` command to show the block-level devices; a USB drive usually 
+   shows up as ``/sdb`` or ``/sdc``
+   (almost never as ``/sda``) and an SD card would usually show up as :file:`/dev/mmcblk0`.  
+   
+   Note: You should specify the whole device you're writing to with 
+   :command:`dd`:  (e.g., :file:`/dev/sdb` or
+   :file:`/dev/mmcblk0`) and **not** just a partition on that device (e.g., :file:`/dev/sdb1` or
+   :file:`/dev/mmcblk0p1`) on that device. 
```

Comment:
I know this was merged, so my recommendation goes to another commit to improve this one. Please consider using full examples or even better let's provide a script. This looks reasonable:

Human readable instructions:
```sh
# check the required uncompressed space
user@host# xz -l ostro.dsk.xz 
Strms  Blocks   Compressed Uncompressed  Ratio  Check   Filename
    1       1     86.8 MiB     87.3 MiB  0.994  CRC64   ostro.dsk.xz

# list available block devices, pick the best that works for you. We need XXGb of space
user@host$ lsblk --list --paths --nodeps --output NAME,HOTPLUG,SIZE
NAME     HOTPLUG        SIZE
/dev/sda       0 32G
/dev/mmcblk0 1 8G

# say you want /dev/mmcblk0

# confirm if it's not mounted (in this case shows no output):
user@host$ findmnt --list --noheading --output SOURCE,TARGET | grep '^/dev/mmcblk0'

# if command above shows no matches, call dd to write image:
user@host$ xzcat ostro.dsk.xz | sudo dd of=/dev/mmcblk0 bs=512K
```

Automated script (host at ostro-os/scripts?): **UNTESTED**
```sh
#!/bin/bash

IMG=${1:?missing image.dsk.xz See https://download.ostroproject.org/releases/ostro-os/}
DEV=${2}

die() {
    echo "ERROR: $*" >&2
    exit 1
}

for c in lsblk findmnt xz; do
    if ! which $c; then
        die "missing '$c' command line utility."
    fi
done

IMG_SIZE=`xz --robot -l "$IMG" | grep -e '^totals'  | cut -f5`

if [ -z "$DEV" ]; then
    echo "NOTE: no device given, available devices are (only hot-pluggable devices displayed):"
    lsblk --paths --list --nodeps --noheadings  --output HOTPLUG,NAME,SIZE
     #| grep -e '^ *1 '
    echo -n "Your choice: "
    read DEV
fi

DEV_SIZE=`lsblk ${DEV} --bytes --list --nodeps --output SIZE | head -n1`

if [ ${DEV_SIZE} -lt ${IMG_SIZE} ]; then
    die "disk ${DEV} size ${DEV_SIZE} bytes is not large enough to fit uncompressed ${IMG} of ${IMG_SIZE} bytes"
fi

MNTS=`findmnt --list --noheading --output SOURCE,TARGET | sed -e 's:^\([^ ]\+\) \+\([^ ]\+\):\1=\2:g' | grep -e "^${DEV}"`
if [ -n "$MNTS" ]; then
    die "disk ${DEV} contains mountpoints: ${MNTS}"
fi

echo "writing ${IMG} to ${DEV}: ${IMG_SIZE} bytes"

xz --verbose --decompress --stdout "${IMG}" | sudo dd of="${DEV}" bs=4M status=progress conv=nocreat,notrunc,fdatasync

if which eject >/dev/null 2>/dev/null; then
    sudo eject ${DEV}
fi
```